src/ValueObject/GalleryItemValueObject.php

```diff
@@ -65,12 +72,15 @@ class GalleryItemValueObject extends ValueObjectBase {
    *   Media source, i.e. the canonical URL to the actual media item.
    * @param string $type
    *   Media item type, either 'image' or 'video'.
+   * @param string $title
+   *   Title for the gallery item.
    * @param string $caption
    *   Caption for the gallery item.
    * @param string $meta
    *   Meta for the gallery item, such as a copyright note.
    */
-  private function __construct(ImageValueObjectInterface $thumbnail, string $source, string $type, string $caption = '', string $meta = '') {
+  private function __construct(ImageValueObjectInterface $thumbnail, string $source, string $type, string $title = '', string $caption = '', string $meta = '') {
+    $this->title = $title;
```

Comment:
Let's put the title as the last parameter to maintain some BC and values don't get set wrongly if someone uses this without a title.